pandas/tseries/timedeltas.py

```diff
@@ -142,13 +137,50 @@ def _validate_timedelta_unit(arg):
 
 
 def _coerce_scalar_to_timedelta_type(r, unit='ns', box=True, errors='raise'):
-    """
-    convert strings to timedelta; coerce to Timedelta (if box), else
-    np.timedelta64
-    """
+    """Convert string 'r' to a timedelta object."""
+
+    try:
+        result = tslib.convert_to_timedelta64(r, unit)
```

Comment:
Not in my implementation and for reasons that I explain <a href="https://github.com/pydata/pandas/pull/13832#discussion_r72812007">below</a>.